content/committees/specification/guide/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Guide to the Jakarta EE Specification Process"
+date: "2021-03-04T00:00:00+00:00"
+layout: "single"
+aliases:
+    - committees/specification/guide/
+---
+
+## Content
+
+- [Introduction]({{< ref "#intro" >}})
+- [Creation Review]({{< ref "#creation" >}})
+- [Plan Review]({{< ref "#plan" >}})
+- [Progress Review]({{< ref "#progress" >}}) *(optional)*
+- [Release Review]({{< ref "#release" >}})
+- [Service Release]({{< ref "#service" >}})
+
+## Introduction {#intro}
+
+This guide is a step-by-step practical guide to help Jakarta EE Specification projects navigate the steps of the Jakarta EE Specification Process ([JESP](https://jakarta.ee/about/jesp/)). 
+It is meant as a helpful tool, so **if in doubt, alway refer to the [JESP](https://jakarta.ee/about/jesp/) for completeness**. Please do not hesitate to contact the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) if you have any questions or comments about anything related to the development of Jakarta EE Specifications.
+
+## Creation Review {#creation}
+
+![Creation Review](JESP_creation-review.png)
+
+* A Jakarta Specification project is created as any other project under the Eclipse Development Process ([EDP](https://www.eclipse.org/projects/dev_process/)).
+* A Jakarta Specification must be approved by the Specification Committee.
+
+### Steps Involved
+
+1. Create a Project Proposal (*).
+2. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[creation review template](https://github.com/jakartaee/specifications/blob/master/creation_review_pr_template.md)**.
+3. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a creation review.
+
+(*) It may be a good idea to socialize the idea with the Specification Committee to solicit support before proceeding with the project creation.
+
+## Plan Review {#plan}
+
+![Plan Review](JESP_plan-review.png)
+
+* Each specification project needs to engage in a Plan Review prior to any extensive development effort in support of a new Specification Version. 
```

Comment:
Is a Plan review required for only Major versions or all Major and Minor versions?  Maybe this statement could be updated to say ".. in support of a new Specification Major or Minor Version"?

Reply:
I also added a line specifically saying that plan reviews are not required for service releases